libtfhe/core/arithmetic/polynomial_torus.cpp

```diff
@@ -114,471 +43,8 @@ double TorusPolynomial<TORUS>::NormInftyDist(
 
     // Max between the coefficients of abs(poly1-poly2)
     for (int32_t i = 0; i < N; ++i) {
-        double r = TorusUtils<TORUS>::normInftyDist(poly1->coefs[i], poly2->coefs[i], zparams);
+        double r = TorusUtils<TORUS>::distance(poly1->coefs + i, poly2->coefs + i, zparams, std::move(alloc));
```

Comment:
you cannot use std::move(alloc) in a loop, it must be createChileAllocator()

(any object on which you call move is in an undefined state after its use. In the case of the allocator, it will be destroyed after move.)